src/test/integration/User.t.sol

```diff
@@ -123,6 +123,16 @@ contract User is Test {
         delegationManager.delegateTo(address(operator), emptySig, bytes32(0));
     }
 
+    /// @dev Undelegate from operator
+    function undelegate() public createSnapshot virtual returns(bytes32){
+        return delegationManager.undelegate(address(this));
+    }
```

Comment:
Mentioned this in another comment, but I'd recommend we have these return `Withdrawal[] memory`

We can query the user's current strategies/shares to figure out what the withdrawal would look like -- i think `delegationManager.getDelegatableShares` should work